src/forms/TimeInput.jsx

```diff
@@ -9,26 +9,33 @@ class TimeInput extends React.Component {
 
 	constructor(props) {
 		super(props);
-		this.state = {
-			value: this.props.value || ''
-		};
+
 		this.onChange = this.onChange.bind(this);
 	}
 
+	/**
+	* @function onChange
+	* @param e Event Object
+	* @description called when the input changes, in turn calls the onChange
+	* 	handler prop, if there is one (eg supplied by redux-form) and an onChangeCallback
+	*	which may be provided by a parent component such as DateTimePicker
+	*/
 	onChange(e) {
-		this.setState({ value: e.target.value });
+		this.props.onChange && this.props.onChange(e.target.value);
 		this.props.onChangeCallback && this.props.onChangeCallback(e.target.value);
 	}
 
 	render() {
 		const {
-			onChangeCallback,	// eslint-disable-line no-unused-vars
 			id,
 			label,
 			name,
 			className,
 			required,
-			value,		// eslint-disable-line no-unused-vars
+			value,
+			error,
+			onChange,			// eslint-disable-line no-unused-vars
```

Comment:
Honestly I'm not sure. 
Right now I know it works when both are preset, but its possible I need only one. 
I'm currently working out the best way these behave with redux form (both within the date picker, or separately). Can I just define the two, and remove in a follow up if I can verify in `mup-web` that only one is actually needed?